tests/src/application/controllers/tasks/load-current-week-tasks-controller.spec.ts

```diff
@@ -38,14 +44,29 @@ describe('LoadCurrentWeekTasksController', () => {
     jest.useRealTimers()
   })
 
+  it('Should call validation with correct value', async () => {
+    const { sut, validationStub } = makeSut()
+    const validationSpy = jest.spyOn(validationStub, 'validate')
+    await sut.handle({ query: { tagId: 'any_tagId' } })
+    expect(validationSpy).toHaveBeenCalledWith({ tagId: 'any_tagId' })
+  })
+
+  it('Should return 400(Not Acceptable) if invalid tagId is provided', async () => {
```

Comment:
400 é bad request e não not acceptable
```suggestion
  it('Should return 400(Bad Request) if invalid tagId is provided', async () => {
```